_data/alumni.yml

```diff
@@ -60,6 +60,22 @@
       icon: github-alt
       name: Github
 
+- name: Amanda Bachmann
+  pic: /img/people/bachmanna.png
+  dates: 2020-present
+  role: Graduate Research Assistant
+  description: "Amanda is a PhD student interested in the nuclear fuel cycle and the best ways to implement nonproliferation safeguards."
```

Comment:
```suggestion
  description: "Amanda was a PhD student working on fuel cycle modeling. She received her PhD in 2023, and is no  postdoc at Argonne National Lab."
```

Reply:
Should say, "is now a postdoc"?

Reply:
Yes. Thank you.

Reply:
Want to link to your PhD?

Reply:
My dissertation? I don't think it's live yet on Ideas

Reply:
Ahh, gottcha! I didn't know how that worked, but fair.